pytket/pytket/utils/expectations.py

```diff
@@ -131,28 +122,22 @@ def get_operator_expectation_value(  # noqa: PLR0912, PLR0913, PLR0915
 
     :param state_circuit: Circuit that generates the desired state
         :math:`\\left|\\psi\\right>`
-    :type state_circuit: Circuit
     :param operator: Operator :math:`H`. Currently does not support free symbols for the
         purpose of obtaining expectation values.
-    :type operator: QubitPauliOperator
     :param backend: pytket backend to run circuit on.
-    :type backend: Backend
     :param n_shots: Number of shots to run if backend supports shots/counts. None will
         force the backend to give the full state if available. Defaults to None
-    :type n_shots: Optional[int], optional
     :param partition_strat: If retrieving shots, can perform measurement reduction using
         a chosen strategy
-    :type partition_strat: Optional[PauliPartitionStrat], optional
     :return: :math:`\\left<\\psi | H | \\psi \\right>`
-    :rtype: complex
     """
     if not n_shots:
         if not backend.valid_circuit(state_circuit):
             state_circuit = backend.get_compiled_circuit(state_circuit)
         try:
-            coeffs: list[complex] = [complex(v) for v in operator._dict.values()]  # noqa: SLF001
```

Comment:
This private member usage can be replaced with [QubitPauliOperator.get_dict()](https://docs.quantinuum.com/tket/api-docs/utils.html#pytket.utils.operators.QubitPauliOperator.get_dict) method.